### PR TITLE
Update README.md to include a basic Docker Compose example

### PR DESCRIPTION
**Pull request by ntalekt**
Nothing much, just a thank you and a simple Docker Compose example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,22 @@ You can optionally attach a configuration file to the container by adding the
 
 **Note**: The recommended restart policy for the container is `on-failed` or `no`
 
+#### Docker Compose Example
+```yaml
+...
```
Comment (jdholtz) [inline README.md:102]: I would just put `services:` here instead of the `...` (and at the end). It makes it easier to copy

File: README.md [hunk 2]
```diff
@@ -97,6 +97,22 @@ You can optionally attach a configuration file to the container by adding the
...
+  auto-southwest:
+    image: jdholtz/auto-southwest-check-in
+    container_name: auto-southwest
+    networks:
```
Comment (jdholtz) [inline README.md:106]: Is `networks:` and `security_opt:` necessary for you? I removed it and it still worked perfectly fine.

File: README.md [hunk 3]
```diff
@@ -97,6 +97,22 @@ You can optionally attach a configuration file to the container by adding the
...
+      - default
+    security_opt:
+      - no-new-privileges:true
+    restart: on-failed
```
Comment (jdholtz) [inline README.md:110]: It actually should be `on-failure`. That's a mistake in the Readme, so it would be great if you could fix it there as well.

**Review by jdholtz — CHANGES_REQUESTED** [review]
Thanks for the contribution! This will be helpful to users. I added a few comments that we can discuss further on.

File: README.md (hunk 1 above)
Comment (ntalekt) [reply README.md]: Addressed in: https://github.com/jdholtz/auto-southwest-check-in/pull/171/commits/146194802d654287272dbfdf9e2a9187d74ee523

File: README.md (hunk 2 above)
Comment (ntalekt) [reply README.md]: You are totally correct and it cleans it up a little. Address in: https://github.com/jdholtz/auto-southwest-check-in/pull/171/commits/c43f49190eb9a96cfbb1f864ba653b94c246bb48

File: README.md (hunk 3 above)
Comment (ntalekt) [reply README.md]: fixed in both, thanks: https://github.com/jdholtz/auto-southwest-check-in/pull/171/commits/3b9fc70f7496d7221ab85ee247c33915c5ecb5c6

**Comment by jdholtz** [comment]
Thanks!

**Pull request merged**